frontend/actions/tagBlue.js

```diff
@@ -0,0 +1,31 @@
+// actions.js
+import { createSlice } from '@reduxjs/toolkit'
+export const BlueTag = createSlice({
+  name: 'bluetag',
+  initialState: {
+    value: [], // list of string: postids 
+  },
+  reducers: {
+    blue: (state, action) => {
+		// Redux Toolkit allows us to write "mutating" logic in reducers. It
+		// doesn't actually mutate the state because it uses the Immer library,
+			// which detects changes to a "draft state" and produces a brand new
+			// immutable state based off those changes
+	    var id = action.payload // value of postid
```

Comment:
suggestion: instead of having three files that do the same thing, instead make it so that the payload will be a string that the tag will consume.